.changeset/hot-chairs-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/shared': patch
+---
+
+Deprecate membershipList in favor of `memberships` that supports paginated responses
```

Comment:
🙃 could you decorate `membershipList` with backticks like you did with `membershipts` word?